pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "crewai-tools"
-version = "0.42.0"
+version = "0.42.1"
```

Comment:
I’m not sure this is as a minor update either… Since we already have a lot of new things in master, it might be a good idea to cut 0.43.0. WDYT?